CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [23.11.0] - 2023-11-23
```

Comment:
I have to admit this is a bit confusing to me. What do you think of removing it again?
Also we should add entries for this PR